odoo/tools/func.py

```diff
@@ -108,6 +108,27 @@ def wrapper(*args, **kwargs):
         return a(b(*args, **kwargs))
     return wrapper
 
+def mockable(func):
```

Comment:
Style: keep two lines between toplevel class/function definitions.

Reply:
Then I add an extra new line above all other method. https://pep8.org/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

Reply:
I added the extra line above all functions ;)